pages/repos/add.vue

```diff
@@ -27,25 +43,26 @@
 </template>
 
 <script setup lang="ts">
-const router = useRouter();
-const githubCookie = useGithubCookie();
-const user = await fetchGithubUser();
 const loading = ref(false);
+const { user } = useAuth();
+const { data: forges } = await useFetch('/api/user/forges', {
+  server: false,
+});
+const selectedForgeId =  ref<number>(Number(localStorage.getItem('forgeId')));
```

Comment:
@aaa006bd The idea was to show a list of forges first and after selecting one you can add repos from it, so this can be `undefined` at the beginning.

Reply:
@anbraten That somehow doesnt work for the github since I cant get into that page. Github shows connected and I cant click on it